ui/src/views/admin/settings/SettingsView.vue

```diff
@@ -1,82 +1,145 @@
 <template>
   <v-container
     id="admin-setting-view"
-    fluid
     tag="section"
   >
     <v-row justify="center">
-      <v-col
-        cols="8"
-        md="8"
-      >
-        <material-card
-          color="primary"
+      <v-col :cols="isMobile? 12 : 8">
```

Comment:
Thank you for the good work!
This is good enough; however some of the elements need fixing so that it will match the design better. 

You can copy the code from UserSettingsView later; I have added a comment with my revised code.